include/deal.II/grid/tria_accessor.h

```diff
@@ -3337,6 +3343,24 @@ class CellAccessor : public TriaAccessor<dim, dim, spacedim>
    * @}
    */
 
+  /**
+   * Return global cell id for an active cell.
+   */
+  types::global_cell_index
+  global_active_cell_index() const;
```

Comment:
Can you add that (assuming this is true) for sequential triangulations, this function returns the same as `active_cell_index()` but that for parallel triangulations, this is not true (and a brief description of what it really is in that case).

Reply:
@peterrum do you still want to augment the documentation somewhat as suggested by @bangerth?